### PR TITLE
FINERACT-1732: Introduce Lombok to bulkimport module

### PR DESCRIPTION
**Pull request by hmg65**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [x] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/data/BulkImportEvent.java [hunk 1]
```diff
@@ -18,19 +18,21 @@
...
+import lombok.Getter;
 import org.apache.fineract.infrastructure.core.domain.FineractContext;
 import org.apache.fineract.infrastructure.core.domain.FineractEvent;
 import org.apache.poi.ss.usermodel.Workbook;
 
+@Getter
 public final class BulkImportEvent extends FineractEvent {
 
-    private final Workbook workbook;
+    private Workbook workbook;
```
Comment (galovics) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/data/BulkImportEvent.java]: Keep the final.

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/domain/FineractEvent.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 public abstract class FineractEvent extends ApplicationEvent implements ContextHolder, Serializable {
 
-    private final FineractContext context;
+    private FineractContext context;
```
Comment (galovics) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/domain/FineractEvent.java]: Keep the final.

**Review by galovics — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/data/BulkImportEvent.java (hunk 1 above)
Comment (hmg65) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/data/BulkImportEvent.java]: done

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/domain/FineractEvent.java (hunk 2 above)
Comment (hmg65) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/domain/FineractEvent.java]: done

**Pull request closed without merging**